src/main/java/com/passbook/sparkeighteen/service/UserService.java

```diff
@@ -54,6 +54,7 @@ public LoginResponse login(final LoginRequest userLogin) throws Exception {
 
         return LoginResponse.builder()
                 .userID(user.getId())
+                .firstname(user.getFirstname())
```

Comment:
This not part of this PR remove it.

Reply:
Updated, As perviously discussed need this to be closed